tests/client/notification_service_tests.cpp

```diff
@@ -9,32 +9,81 @@
 #include <string>
 
 #include <boost/asio.hpp>
+#include <boost/bind/bind.hpp>
 #include <chrono>
 #include <boost/date_time/posix_time/posix_time.hpp>
 
 namespace xpx_chain_sdk::tests {
 
 #define TEST_CLASS NotificationService
 
-    boost::asio::io_context io;
-    boost::asio::deadline_timer timer(io, boost::posix_time::seconds(60));
-
     ClientData clientData;
     auto client = xpx_chain_sdk::getClient(std::make_shared<xpx_chain_sdk::Config>(getTestConfiguration()));
     auto account = getTestAccount(clientData.privateKey);
 
+    void wait_Block(const boost::system::error_code&, int* count, bool* isReceived, boost::asio::deadline_timer* timer){
```

Comment:
camelCase please = )

Reply:
For camelCase, I just heard about this. Does that means I need to change wait_Block() to waitBlock() instead ?